.travis.yml

```diff
@@ -41,20 +29,24 @@ matrix:
       language: generic
       python:
       osx_image: xcode7.3
+    - os: windows
+      language: shell
+      before_install:
+        - choco install python2 -x86
+        - ln -s /c/Python27/python.exe /c/Python27/python2.7.exe
+      env:
+        - PATH="/c/Python27:/c/Python27/Scripts:$PATH"
+        - TEST_SUITE="$PYTEST"
+    - os: windows
+      language: shell
+      before_install:
+        - choco install python2 -x86
+        - ln -s /c/Python27/python.exe /c/Python27/python2.7.exe
+      env:
+        - PATH="/c/Python27:/c/Python27/Scripts:$PATH"
+        - TEST_SUITE="./etc/release/release.sh"
 
-install:
-  - ./configure
 
-script:
-  - $TEST_SUITE
 
-notifications:
-  irc:
-    channels:
-      - "chat.freenode.net#aboutcode"
-  on_sucess: change
-  on_failure: always
-  use_notice: true
-  skip_join: true
-  template:
-    - "%{repository_slug}#%{build_number} (%{branch}-%{commit}:%{author})-%{message}- %{build_url}"
```

Comment:
Why remove this?

Reply:
actually I was trying to sort out the error in Travis CI ,I am currently working over it.